dwave/system/temperatures.py

```diff
@@ -482,13 +482,13 @@ def fast_effective_temperature(sampler=None, num_reads=None, seed=None, T_guess
         https://www.jstor.org/stable/25464568
     
     Examples:
-       Draw samples from the default DWaveSampler(), and establish the temperature
+       Draw samples from a :class:`~dwave.system.samplers.DWaveSampler`, and establish the temperature
        
        >>> from dwave.system.temperatures import fast_effective_temperature
        >>> from dwave.system import DWaveSampler
        >>> sampler = DWaveSampler()
        >>> T = fast_effective_temperature(sampler)
-       >>> print('Effective temperature at freeze-out is',T)
+       >>> print('Effective temperature at freeze-out is',T)    # doctest: +SKIP
```

Comment:
```suggestion
       >>> print('Effective temperature at freeze-out is',T)    # doctest: +SKIP
       0.21685104745347336
```

This use of `# doctest: +SKIP` is needed because the mock sampler returns 0. The number in the result is measured for  Advantage_system4.1